src/Utilities/Compiler.CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -2,16 +2,16 @@
 
 using System.Linq;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
 
 namespace Analyzer.Utilities.Extensions
 {
     internal static partial class SyntaxNodeExtensions
     {
         public static SyntaxNode WalkDownParentheses(this SyntaxNode node)
         {
-            const int parenthesizedExpressionRawKind = 8632;
             SyntaxNode current = node;
-            while (current.RawKind == parenthesizedExpressionRawKind && current.ChildNodes().FirstOrDefault() is SyntaxNode expression)
+            while (current.IsKind(SyntaxKind.ParenthesizedExpression) && current.ChildNodes().FirstOrDefault() is SyntaxNode expression)
```

Comment:
This was in language-agnostic layer, but the code is specific for C# and won't walk down parentheses for VB. I moved it to the new C#-specific project to avoid accidental use of this extension method in VB-specific code.